gluco-check-common/strings/en-US/terms.md

```diff
@@ -0,0 +1,39 @@
+# Gluco Check
+
+## _Health information disclaimer_
+Gluco Check can only provide you with general info to help you monitor your activities.
+While we hope this information will be useful to you, remember that it can never 
+replace advice from a medical professional. Talk to your doctor if you have questions 
+about treatment!
+
+## _Privacy Policy_
+This document describes the information being collected by  
+Gluco Check, https://github.com/nielsmaerten/gluco-check (the "Service" or the "Software") and how we use this information.
+
+#### What information is collected?
+When you sign in to the Service, we ask you for the URL of your *Nightscout site* and a *token*.
+We store this along with the *email address* of your Google Account.
+Now when you ask Gluco Check a question, we know whose site to check.
+
+
+#### How is this information used?
+When you invoke Gluco Check through Google Assistant,
+we check if there's a Nightscout site associated with your Google Account.
+If so, we gather the information you requested from your Nightscout and read it out to you.
+We do not store the information we collect from your Nightscout. Not even in logs.
+Our logs do include the question you asked Gluco Check and when.
+We use this information to understand how the Service is used, and to diagnose issues.
+
+#### Sharing of information
+We don't share your information with third parties.
+Only the 2 administrators of Gluco Check (Niels Maerten & David D'Amico) have access.
+
+#### Cookies
+We use Google Analytics to collect anonymous information about visitors to our website.
```

Comment:
Ah yes. Analytics are very useful to better understand how people find us.
(which search terms they use, who's referring to us, etc)  
It would definitely be nice to have analytics integrated when we launch

Reply:
@ddamico  Our analytics tag is: UA-68993648-4

It's possible this will change when we move to a new domain, so probably sensible to pull this from the .env file